plugins/algoltek-usbcr/fu-algoltek-usbcr-device.h

```diff
@@ -8,6 +8,8 @@
 
 #include <fwupdplugin.h>
 
+#define ALGOLTEK_USBCR_VENDOR_ID 0x58f
```

Comment:
I think move this to fu-algoltek-usbcr-device.c, give it a FU_ prefix then LGTM. Thanks.